Formula/y/yewtube.rb

```diff
@@ -130,8 +130,11 @@ def caveats
   end
 
   test do
-    Open3.popen3("#{bin}/yt", "/Drop4Drop x Ed Sheeran,", "d 1,", "q") do |_, _, stderr|
-      assert_empty stderr.read, "Some warnings were raised"
-    end
+    system bin/"yt",
+      "set checkupdate false,",
+      "set ddir \"#{testpath}\",",
+      "/youtube-dl test video,", "d 1,", "q"
+    downloaded_file = Dir.glob(testpath/"mps/*").first
+    assert_match(/^(WebM)|(.*MP4.*)$/, shell_output("file --brief \"#{downloaded_file}\""))
```

Comment:
I wasn't getting consistent results in setting an explicit format type for the download, so here we're just testing whether it's a webm or an mp4.